exomiser-cli/README.md

```diff
@@ -184,42 +203,61 @@ localhost/exomiser-cli           latest           c12b1878a8f3  52 years ago
 localhost/exomiser-cli           ${project.version} c12b1878a8f3  52 years ago    273 MB
 ```
 
-To run the image you will need the standard exomiser directory layout to mount as separate volumes as in the cli and
-supply an `application.properties` file or environmental variables to point to the data required _e.g._
+### Activate the docker image
 
-```shell
-docker run -v "/data/exomiser-data:/exomiser-data" \
- -v "/opt/exomiser/exomiser-config/:/exomiser"  \
- -v "/opt/exomiser/exomiser-cli-${project.version}/results:/results"  \
- localhost/exomiser-cli:${project.version}  \
- --analysis /exomiser/test-analysis-exome.yml  \
- --vcf /exomiser/Pfeiffer.vcf.gz  \
- --spring.config.location=/exomiser/application.properties
-```
-
-or using Spring configuration arguments instead of the `application.properties`:
```

Comment:
We still need these docs for the default distroless image. I'd suggest splitting the docs into `Docker - distroless` and `Docker bash`